### ISSUE TITLE
Sprint 1 - Tristan's Goals

### ISSUE THREAD

**Issue by kh3dron** [CNSabers/bscs2016#16]
-Fully learn how Github works - well enough to help other people (with merge conflicts, stuff that comes up like that)

-I think I can pick CSS back up fairly quickly (did a codecademy course over the summer, I get how it works), so I'll do that

-Learn to write better HTML from memory - I'm not sure if mine is bad, or if HTML as a language is fundamentaly less elegant than, say, Python. It seems to be, but in either case, I should get better at writing it to the point where I don't need to have an example text next to me all the time.

**Comment by shickey** [comment]
Tristan, how did sprint 1 go re: these goals? Update this ticket with any thoughts, evidence, etc.!

**Comment by kh3dron** [comment]
I didn't learn as much about Github as I wanted to, but I still did learn some. I understand how to make branches and do commits, so enough to use it properly. That's all I needed to be able to do, anyway. 

I really know how CSS works now. I know how to choose what I want to style, and I can figure out what each variable does by searching it. 

My HTML is also better. It was mostly just getting back into it after a few months that made it easier. I also started a Lynda tutorial that helped- it wasn't even really teaching me how to write the code, it was just watching someone do it that made it easier. I'd say I did pretty well for these goals.
